src/components/entries/FEEL/Feel.js

```diff
@@ -37,7 +35,7 @@ import Tooltip from '../Tooltip';
 
 const noop = () => {};
 
-function FeelTextfieldComponent(props) {
+function FeelTextfield(props) {
```

Comment:
:spiral_notepad:  The old context is removed, interactions with the popups are now done via the eventbus. I also cleaned up formatting a tiny bit, and removed some popup specific helpers from this class. Also removed the decorator pattern, as it was too extra.